static/app/components/events/interfaces/crashContent/exception/actionableItems.tsx

```diff
@@ -442,15 +442,14 @@ export function ActionableItems({event, project}: ActionableItemsProps) {
       )
   );
 
-  for (const errorKey in Object.keys(errorMessages)) {
+  for (const errorKey of Object.keys(errorMessages)) {
     const isWarning = ActionableItemWarning.includes(
       errorKey as ProguardProcessingErrors | NativeProcessingErrors | GenericSchemaErrors
     );
     const shouldDelete = hasErrorAlert ? isWarning : !isWarning;
 
     if (shouldDelete) {
-      // @ts-expect-error TS(7053): Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
-      delete errorMessages[errorKey];
+      delete errorMessages[errorKey as keyof typeof errorMessages];
```

Comment:
note: I’m pretty sure this can’t have worked before. If we iterate with:

```
for (const errorKey in Object.keys(errorMessages))
```

`errorKey` will be the stringified index of the keys array, so `['0', '1']` etc. Then, going to the actual `errorMessages` object and deleting something at `'0'` doesn't do anything unless the key is literally the string `'0'`.

Maybe the intention was to do:

```
for (const errorKey in errorMessages)
```

to iterate over all keys?

@roggenkemper am I missing something here?

Reply:
good catch! i guess the case that it covered wasn't common enough to come up often.